scss/_modal.scss

```diff
@@ -80,20 +106,27 @@
   display: flex;
   flex-direction: column;
   width: 100%; // Ensure `.modal-content` extends the full width of the parent `.modal-dialog`
-  padding: $modal-content-padding;
-  color: $modal-content-color;
+  padding: $modal-content-padding; // Boosted mod
```

Comment:
CSS var?

Reply:
See https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1181/commits/88deeeced3c1e7fe08118b1b19d0c84450067e78 in order to resolve the comments regarding `$modal-*`